source/scripts/app.js

```diff
@@ -503,9 +503,19 @@ function homeSearchFunction() {
       e.preventDefault();
       // store search string and navigate to explore page
       // let searchQuery = e.target.value;
+      let searchQuery = e.target.value;
+
+      //clear search and route to explore
+      e.target.value = "";
       router.navigate("explore-page");
 
       // TODO more here once explore page setup
+      //display results of search
+      let explore = document.querySelector("explore-page");
+      let shadow = explore.shadowRoot;
+      let bar = shadow.getElementById("search-bar");
+      bar.value = searchQuery;
```

Comment:
Since it's now an object, we have to change this line
```suggestion
      bar.value = searchQuery.query;
```